### PR TITLE
feat: add template option

### PR DESCRIPTION
**Pull request by xiaofan2406**
add `template` options for specifying `--create` flag to use for generating new migration files

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -33,58 +33,46 @@ Index.prototype.run = function() {
...
-    , '};'
-    , ''
-    , 'exports.down = function(next) {'
-    , '  next();'
-    , '};'
-    , ''
-  ]).join('\n');
+Index.prototype.create = function () {
+  var template =
+    this.opts.template ||
```
Comment (xiaofan2406) [inline lib/index.js:52]: this is the main change here.

all other changes are caused by my editors `prettier` setup.

File: README.md [hunk 2]
```diff
@@ -23,6 +20,7 @@ var Migrations = require('migrations'),
 module.exports = new Migrations({
   dir: __dirname + '/migrations', // directory with migration files
   meta: new MetaFile({path: __dirname + '/migrations.json'}) // meta information storage
+  template: '', // template used by `--create` flag to generate a new migration file
```
Comment (xiaofan2406) [inline README.md:23]: usage of this change.

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "migrations",
-  "version": "0.1.8",
+  "version": "0.1.7",
```
Comment (xiaofan2406) [inline package.json:3]: not sure how this was 0.1.8

Comment (bchew) [reply package.json:3]: Need to confirm with @vinteo, either it as deliberate which would mean the package-lock.json should be updated instead. This is the npm package which we could potentially try and get access back again.. https://www.npmjs.com/package/migrations

Comment (vinteo) [reply package.json:3]: I pushed it up before, it just means we haven't published it yet

**Comment by adityasabnis** [comment]
Merging 🚀 🚀

**Pull request merged**